features/fields-fout-cases.feature

```diff
@@ -141,14 +140,18 @@ Rule: De Fields parameter mag maximaal 130 veld paden bevatten
     | code     | name   | reason                          |
     | maxItems | fields | Array bevat meer dan 130 items. |
 
+    Voorbeelden:

```

Comment:
Dit voorbeeld begrijp ik niet. Neem je hier in fields nu 131 keer een waarde voor een aNummer op?
Dat is toch niet hetzelfde als een pad? Volgens mij moet dit scenario betrekking hebben op een fields met 131 valide paden. Daarvoor zijn denk ik wel 131 verschillende properties in de OAS nodig en de vraag is of we dat halen.
Of misschien kun je in de fields parameter paden herhalen.

Reply:
Thanks, ik heb nu paden opgenomen.

Reply:
Prima